addons/mail/i18n/mail.pot

```diff
@@ -134,6 +134,18 @@ msgstr ""
 msgid "%(name)s: %(message)s)"
 msgstr ""
 
+#. module: mail
+#. odoo-python
+#: code:addons/mail/models/discuss/discuss_channel.py:0
+msgid ""
+"%(new_line)s%(new_line)sType %(bold_start)s@username%(bold_end)s to mention "
+"someone, and grab their attention.%(new_line)sType "
+"%(bold_start)s#channel%(bold_end)s to mention a channel.%(new_line)sType "
+"%(bold_start)s/command%(bold_end)s to execute a command.%(new_line)sType "
+"%(bold_start)s:shortcut%(bold_end)s to insert a canned response in your "
+"message."
+msgstr ""
```

Comment:
Aren't those from another PR?

Reply:
Not yet, the other PR adds `::shortcut` and `:emoji`, which isn't the case here.
Don't know why this is new in the POT generation.

Reply:
I guess non-exported terms from other PRs then.